framework/helpers/BaseArrayHelper.php

```diff
@@ -625,6 +625,21 @@ public static function isIn($needle, $haystack, $strict = false)
     }
 
     /**
+     * Checks whether a variable is an array or [[\Traversable]].
+     *
+     * This method does the same as the PHP function [is_array()](http://php.net/manual/en/function.is-array.php)
+     * but it does not only work for arrays but also objects that implement the [[\Traversable]] interface.
```

Comment:
but it does not only work for arrays but also objects that implement the → but additionally works for objects that implement the